ranger/config/rifle.conf

```diff
@@ -178,6 +178,7 @@ ext mobi, has ebook-viewer, X, flag f = ebook-viewer -- "$@"
 ext cb[rz], has qcomicbook, X, flag f = qcomicbook "$@"
 ext cb[rz], has mcomix,     X, flag f = mcomix -- "$@"
 ext cb[rz], has zathura,    X, flag f = zathura -- "$@"
+ext cb[rz], has atril,    X, flag f = atril -- "$@"
```

Comment:
Is this MATE's atril? That looks like a much more general document viewer so maybe it should be added for pdf and djvu and other formats it supports too?

Reply:
i only put in what i tested

Reply:
atril with pdf and djvu is already in.....

Reply:
You're right, I didn't check closely enough.